src/persist-types/src/columnar.rs

```diff
@@ -259,6 +259,28 @@ pub trait Schema<T>: Debug + Send + Sync {
     fn encoder(&self, cols: ColumnsMut) -> Result<Self::Encoder, String>;
 }
 
+/// A __stable__ encoding for a type that gets durably persisted in an
+/// [`arrow::array::FixedSizeBinaryArray`].
+pub trait ColumnarCodec<T>: Debug + PartialEq + Eq + PartialOrd + Ord {
```

Comment:
:100:

The name is a touch general for what it is. I'm okay with it, but we could also consider something like FixedSizedCodec

Reply:
Yeah I bounced around a bit on a name, I like `FixedSizeCodec` though, updated!